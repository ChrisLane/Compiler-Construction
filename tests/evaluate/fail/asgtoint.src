// We should only be able to assign to an identifier. This should not evaluate.
main() {
    10 = 20;
}
