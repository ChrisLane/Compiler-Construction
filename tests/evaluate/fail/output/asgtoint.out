Fatal error: exception Failure("Cannot evaluate unsupported expression.")
